.github/workflows/ci.yml

```diff
@@ -13,6 +13,14 @@ jobs:
     name: Test Forge / Unit and Integration Tests
     runs-on: ubuntu-latest
     steps:
+      - name: Set NODE_URL for foundry-rs/starknet-foundry
+        if: ${{ github.event.pull_request.head.repo.full_name == 'foundry-rs/starknet-foundry' }}
+        run: echo "NODE_URL=$(echo ${{ secrets.NODE_URL }}:7070)" >> $GITHUB_ENV
```

Comment:
shouldn't info about port be part of the env var?

Reply:
`sncast` might want to use this variable in the future, but with a different port (integration node). We could create many secrets/env vars, but then it would require different parsing in tests and higher maintenance. I'm not happy with either choice

cc @THenry14

Reply:
I mean, if they want to use different port, it's better not to include it here, and i'd rather have 2 different variables for 2 different endpoints than hardcoding this (which is not allowing different ports inside our tests as well)

Reply:
Honestly I'd leave this as is, it will be easier to maintain than multiple secrets. Our IP won't change, probably ever, ports have changed from time to time.

Reply:
I don't get it. Since the ports will change in the future, why hardcode them? It's easier to update the GH env var than to make a PR and have it merged, isn't it?

Reply:
I find to harder to be honest :D for example, not every developer may be able to do so due to permissions, but every dev will be able to open a PR. Moreover it is very clear what is being used at a glance here (is INTEGRATION_NODE_URL 7070 or 7171?). I don't kow, up to you really since both approaches are fine, but personally Id probably leave this as is.